scripts/datasets/mhp_v1.py

```diff
@@ -0,0 +1,48 @@
+"""Prepare Multi-Human Parsing V1 dataset"""
+import os
+import shutil
+import argparse
+import zipfile
+from gluoncv.utils import download, makedirs
+from googleDriveFileDownloader import googleDriveFileDownloader
```

Comment:
`googleDriveFileDownloader` may not be a default library, users have to install it. Please follow the example here to write a `try_import_ gdfDownloader()` and use it here. In this case, users don't have to install this library when they install gluoncv. 

https://github.com/dmlc/gluon-cv/blob/master/gluoncv/utils/filesystem.py#L43